ui/src/views/Profile/ProfileView.tsx

```diff
@@ -62,6 +68,17 @@ export default function ProfileView(): React.ReactElement {
     mode: "onChange",
   });
 
+  React.useEffect(() => {
```

Comment:
What do you want to do here ? I think this won't be stable. Let's have a talk about that

Reply:
It is to set the status to Online, we will have to do that for every url. In what way will this not be stable?

Reply:
I am actually not sure anymore, we can run some tests together later on